vpn-router-web/src/main/java/vpnrouter/web/ClientWebView.java

```diff
@@ -0,0 +1,38 @@
+package vpnrouter.web;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class ClientWebView {
+    private String ipAddress;
+    private String name;
+    private boolean tunnelled;
+
+    @Override
+    public int hashCode() {
```

Comment:
Do methods `hashCode` and `equals` behave the same way as if there were `@EqualsAndHashCode(of = "ipAddress")` annotation on the class? (I, for example, don't know because it's difficult to understand this code 🤦 ) If yes, why do you make a reader of this code suffer? 😄 Why write a lot of complex code instead of a single short annotation?

Reply:
You're absolutely right) The behavior is equivalent of using `@EqualsAndHashCode(of = "ipAddress")`. Believe me, I didn't intend to make you suffer, I just forgot about this annotation 😅